FilmFestivalLoader/Tests/test_planner_interface.py

```diff
@@ -0,0 +1,53 @@
+import tempfile
+import unittest
+
+from Shared.planner_interface import FestivalData, Section
+
+
+class SectionsTestCase(unittest.TestCase):
+    def setUp(self):
+        self.temp_dir = tempfile.TemporaryDirectory()
+        self.festival_data = FestivalData(self.temp_dir.name)
+
+    def tearDown(self):
+        self.temp_dir.cleanup()
```

Comment:
Please consider to not leave the destruction of the temp dir to the garbage collector, but use `shutil.rmtree()`.

Reply:
I find `rmtree()` a bit scary...
Plus: we will need a `try` - `finally` around it, making the code (superfluously) more complicated.

So, I considered but won't fix.